src/Pagination/Pagination.component.tsx

```diff
@@ -0,0 +1,268 @@
+// VENDOR
+import * as React from 'react';
+import classNames from 'classnames';
+import styled from '@xstyled/styled-components';
+// ANCHOR
+import { ButtonProps } from '../Button/Button.component';
+import { Goto } from './Goto';
+import {
+    Button as AnchorButton,
+    ChevronLeft,
+    ChevronRight,
+    Ellipses,
+    Typography,
+} from '..';
+
+// Constrain a value between a min and max
+const constrain = (min: number, value: number, max: number) =>
+    Math.min(Math.max(value, min), max);
+
+const Button = ({
+    variant = 'minimal',
+    size = 'sm',
+    outline = false,
+    ...props
+}: ButtonProps) => (
+    <AnchorButton variant={variant} size={size} outline={outline} {...props} />
+);
+
+type Size = 'sm' | 'xs';
+type Variant = 'pager' | 'minimal';
```

Comment:
This is a very small quibble, but I feel that `pager` may not be the best wording as technically they're both still pagers. Maybe `default` and `minimal`?

Reply:
`default` is actually what I originally called it, but then I changed it because, what if we no longer want it to be the default option? I agree that `pager` isn't great, but honestly I have no idea what else to call it.

Reply:
That's fair. My other thought was `buttons`, since one view is using buttons and the other is... not. ;)